ras_party/controllers/account_controller.py

```diff
@@ -269,13 +274,13 @@ def change_respondent_password(token, payload, tran, session):
 def request_password_change(payload, session):
     v = Validator(Exists('email_address'))
     if not v.validate(payload):
-        raise RasError(v.errors, 400)
+        raise ClientError(v.errors, 400)
 
     email_address = payload['email_address']
 
     respondent = query_respondent_by_email(email_address, session)
     if not respondent:
-        raise RasError("Respondent does not exist.", status=404)
+        raise ClientError("Respondent does not exist", status=404)
 
     logger.debug("Requesting password change", party_id=respondent.party_uuid)
```

Comment:
This appears in the log as:
```
{"party_id": "UUID('b73ad0e0-c83d-454b-93ac-12ed58c741e2')", "event": "Requesting password change", ... }
```
We should wrap `respondent.party_uuid` with `str` in all places where we're accessing the uuid from the database object.